topics:
  inputs:
    trajectory:
      name: /arm/trajectory
      type: arm_planner/ArmTrajectory
  outputs:
    joint_state:
      name: /arm/joint_states
      type: sensor_msgs/JointState